scripts/globals/items/burning_cesti.lua

```diff
@@ -12,7 +13,7 @@ require("scripts/globals/magic");
 -----------------------------------
 
 function onAdditionalEffect(player,target,damage)
-    local chance = 5;
+    local chance = 10;
```

Comment:
Unless you got a source verifying proc and dmg, please don't change existing proc and dmg rates